deployments/daemon-set/nginx-plus-ingress.yaml

```diff
@@ -1,43 +0,0 @@
-apiVersion: extensions/v1beta1
```

Comment:
man, you got carried away :)

Reply:
please make the changes here similar to the changes to the deployment file

Reply:
Oops, yeah I probably missclicked, recovering these